integrations/data/TestStream_Derivative.srpl

```diff
@@ -6,7 +6,7 @@ rpname
 packets value=1001 0000000002
 dbname
 rpname
-packets value=1002 0000000003
+packets value2=0 0000000003
```

Comment:
This, and the other brpl changes was to test for the issue in [this comment](https://github.com/influxdata/kapacitor/issues/1329#issuecomment-295558255)